app/views/static/deploy-provider-details-existing-vms.html.haml

```diff
@@ -0,0 +1,137 @@
+%div{"ng-controller" => "containers.deployProviderDetailsExistingVMsController"}
+  %form.form-horizontal{"ng-if" => "data.provisionOn == 'existingVms'"}
+    .form-group
+      %label.miq-input-label-class.miq-input-label-class-lg
+        = _("Private SSH Key")
+      .miq-input-class
+        %input#deploy-key.form-control{"ng-change" => "validateForm()", "ng-model" => "data.deploymentKey", :required => "", :type => "password"}/
+      %a.miq-clear-input-button{"ng-click" => "clearDeploymentKey()"}
+        %span.fa.fa-close{"aria-hidden" => "true"}
+      %button{"ng-click" => "browseKeyFile()", :type => "button"}
+        = _("Browse")
+      %div{:style => "display: none;"}
+        %input#browse-key-input{:type => "file"}
+    %div{"pf-form-group" => "", "pf-input-class" => "miq-input-class", "pf-label" => _("Username"),
+    "pf-label-class" => "miq-input-label-class miq-input-label-class-lg", :required => ""}
+      %input#deploy-username{"ng-change" => "validateForm()", "ng-model" => "data.deploymentUsername", :required => "", :type => "text"}/
+  .miq-page-text
+    = _("Add tags individually for each VM or select multiple rows and edit tags as a group.")
+  .nodes-table-container.add-nodes-table
+    %div{:config => "toolbarConfig", "pf-toolbar" => ""}
+      .include-actions
+        %span.dropdown.primary-action{"auto-close" => "outsideClick", :dropdown => "", "is-open" => "editRolesStatus.open"}
+          %button.btn.btn-default.dropdown-toggle{"dropdown-toggle" => "", "ng-click" => "onToolbarMenuShow()",
+          "ng-disabled" => "disableMasterNodeActions", :type => "button"}
+            = _("Edit Roles")
+            %span.caret
+          .dropdown-menu{:role => "menu"}
+            .dropdown-container
+              %a.select-all-roles.btn-link{"ng-click" => "toolbarMenuSelectAll()"} Select All
+              %label.checkbox
+                %input{"ng-model" => "toolbarMenu.master", :role => "presentation", :type => "checkbox"}
+                  Master
```

Comment:
@himdel not sure I understand what you meant here. these are checkboxes, therefore should be input, right? 
added the translation (requires a new line), thanks.